include/cv_camera/capture.hpp

```diff
@@ -0,0 +1,278 @@
+// Copyright [2015] Takashi Ogura<t.ogura@gmail.com>
```

Comment:
Why did you change header files' extension? this makes harder to know what we changed with respect to the original repo. If there is not a good reason to do this I would suggest keeping the original `.h` files